pkg/action/testdata/scan_archive

```diff
@@ -1070,15 +1081,15 @@
                     "RuleName": "go_file_read"
                 },
                 {
-                    "Description": "renames files",
+                    "Description": "rename",
```

Comment:
this description change seems off?

Reply:
I think it matched a different rule with the changes. I can double-check.

Reply:
Fixed in [`a51618e` (#898)](https://github.com/chainguard-dev/malcontent/pull/898/commits/a51618ef6aa1a382d92131da3a533c74fd5b7403) and [`2e0fd4a` (#898)](https://github.com/chainguard-dev/malcontent/pull/898/commits/2e0fd4a37854f78eaab9fc4b3c33f0c5e0131410).